packages/account/src/Sections/Security/Passwords/change-password-form.jsx

```diff
@@ -0,0 +1,167 @@
+            action: 'redirect_to_login',
+            header: 'password_changed',
+        };
+        const search_params = new URLSearchParams(params).toString();
+
+        history.push({
+            pathname: routes.root,
+            search: `?${search_params}`,
+        });
+    };
+
+    const onSubmit = (values, { setSubmitting, setStatus, resetForm }) => {
+        setStatus({ msg: '' });
+        setIsBtnLoading(true);
+        if (is_trading_password) {
+            WS.tradingPlatformPasswordChange(values).then(data => {
+                if (isMounted()) {
+                    setIsBtnLoading(false);
+                    if (data.error) {
+                        setStatus({ msg: data.error.message });
+                        setSubmitting(false);
+                    } else {
+                        setIsSubmitSuccess(true);
+                        resetForm({ new_password: '' });
+                        setTimeout(() => {
+                            if (isMounted()) {
+                                setIsSubmitSuccess(false);
+                                setSubmitting(false);
+                            }
+                        }, 3000);
+                    }
+                }
+            });
+        } else {
+            WS.authorized.storage.changePassword(values).then(data => {
+                if (isMounted()) {
+                    setIsBtnLoading(false);
+                    if (data.error) {
+                        setStatus({ msg: data.error.message });
+                    } else {
+                        setIsSubmitSuccess(true);
+                        logout().then(handlePasswordChange);
+                    }
+                    setSubmitting(false);
+                }
+            });
+        }
+    };
+
+    const validateFields = values => {
+        setIsSubmitSuccess(false);
+        const errors = {};
+
+        const required_fields = ['old_password', 'new_password'];
+        required_fields.forEach(required => {
+            if (!values[required]) errors[required] = localize('This field is required');
+        });
+
+        if (values.new_password) {
+            if (!validLength(values.new_password, { min: 8, max: 25 })) {
+                errors.new_password = localize('Password length should be between 8 to 25 characters.');
+            }
+            if (values.old_password === values.new_password) {
+                errors.new_password = localize('Current password and new password cannot be the same.');
+            }
+            if (values.new_password.toLowerCase() === email.toLowerCase()) {
+                errors.new_password = localize('Your password cannot be the same as your email address.');
+            }
+            if (!validPassword(values.new_password)) {
+                errors.new_password = getErrorMessages().password();
+            }
+        }
+
+        return errors;
+    };
+
+    return (
+        <React.Fragment>
+            <Formik
```

Comment:
This is just an optional suggestion: using `useFormik` hook instead of the old `Formik` component that uses render prop technique which is not recommended anymore.
This is also true for some other Formik use cases in this PR. I don't repeat this for other ones